src/System.IO.FileSystem/tests/File/ReadAllBytes_Str.cs

```diff
@@ -0,0 +1,161 @@
+        Byte[] bits;
+        Byte[] rbits;
+        int count;
+        //		FileStream stream;
+        //		BinaryReader reader;
+
+
+        try
+        {
+            //Argument 
+
+            strLoc = "Loc_001oo";
+
+            iCountTestcases++;
+
+            path = null;
+            try
+            {
+                rbits = File.ReadAllBytes(path);
+
+                iCountErrors++;
+                Console.WriteLine("Err_3947sgd! Expected Exception here.");
+            }
+            catch (ArgumentNullException)
+            {
+            }
+            catch (Exception ex)
+            {
+                iCountErrors++;
+                Console.WriteLine("Err_9734sfg! Unexoected exception thrown; {0}", ex.GetType().Name);
+            }
+
+            //@TODO Non-existent
+
+            //Vanilla - make sure that the normal works!
+
+            strLoc = "Loc_002oo";
+
+            iCountTestcases++;
+
+            path = Path.GetTempFileName();
+            bits = new Byte[0];
+            File.WriteAllBytes(path, bits);
+            rbits = File.ReadAllBytes(path);
+            if (rbits.Length != 0)
+            {
+                iCountErrors++;
+                Console.WriteLine("Err_3456gsd! Wrong value returned. {0}", rbits.Length);
+            }
+
+            count = 200;
+            bits = new Byte[count];
+            for (int i = 0; i < count; i++)
+                bits[i] = (byte)i;
+            File.WriteAllBytes(path, bits);
+            rbits = File.ReadAllBytes(path);
+            for (int i = 0; i < count; i++)
+            {
+                if (rbits[i] != i)
+                {
+                    iCountErrors++;
+                    Console.WriteLine("Err_93457gs_{0}! Wrong value returned. {1}", i, rbits[i]);
+                }
+            }
+
+            //Writing again overwrites
+            count = 20;
+            bits = new Byte[count];
+            for (int i = 100; i < 100 + count; i++)
+                bits[i - 100] = (byte)i;
+            File.WriteAllBytes(path, bits);
+            rbits = File.ReadAllBytes(path);
+            if (rbits.Length != count)
+            {
+                iCountErrors++;
+                Console.WriteLine("Err_34sg! Wrong value returned. {0}", rbits.Length);
+            }
+            for (int i = 100; i < 100 + count; i++)
+            {
+                if (rbits[i - 100] != i)
+                {
+                    iCountErrors++;
+                    Console.WriteLine("Err_93457gs_{0}! Wrong value returned. {1}", i, rbits[i]);
+                }
+            }
+
+            if (File.Exists(path))
+                File.Delete(path);
+
+            //large values
+            path = Path.GetTempFileName();
+            count = 1 << 24;
+            while (count > 0)
+            {
+                try
+                {
+                    bits = new Byte[count];
+                    break;
+                }
+                catch (OutOfMemoryException)
```

Comment:
Doing so doesn't significantly impact the runtime, planning on leaving as is unless there's another reason to.